.github/workflows/snyk.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Clojure Snyk Check
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
curious what the other types are for, since this sample only has `labeled`: https://github.com/puppetlabs/security-snyk-clojure-action/blob/main/public_sample_workflow.yaml#L6

Reply:
So, I was finding that specifying `labeled` here replaced the default triggers (which are the others in this list), rather than added to them. This meant that if, for example, I added the label while opening the PR, the action wasn't triggering. Likewise, once the label was on, if you pushed to the branch, it wouldn't retrigger. So my conclusion is that putting `labeled` in this list means it only triggers in response to a "labeling" event.

The conditional in the action proper prevents it from being run when opened, reopened, or synchronized if the label is not present. The action shows up in the list as "skipped".

Reply:
gotcha, thanks!

When the action is "skipped", does it show as failed? If not, I could see us forgetting to add the label.

Reply:
I don't think so. It shows up as greyed out though in the list. That would be enough for me to be like, "ew no we need the label."

Reply:
ah okay, greyed out seemed fine. just as long as it's not like, "All checks have passed"!, since I definitely wouldn't notice if, like, it had 2 checks instead of 3 or something.